src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp

```diff
@@ -166,6 +163,13 @@ void WellSolverBase::implicitStepSetup( real64 const & time_n,
   } );
 }
 
+void WellSolverBase::implicitStepComplete( real64 const & time_n,
+                                           real64 const & dt,
+                                           DomainPartition & domain )
+{
+  GEOSX_UNUSED_VAR( time_n, dt, domain );
+}
+
```

Comment:
Yes, for now I moved this function to the hpp file to have something more concise:
https://github.com/GEOSX/GEOSX/blob/26b65c4e44ff73cd0f44f341da402f50e5f669f0/src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.hpp#L151-L153

If no one has objections, I could also change directly the function in `SolverBase` to:
```
void SolverBase::implicitStepComplete( real64 const & GEOSX_UNUSED_PARAM( time ),
                                       real64 const & GEOSX_UNUSED_PARAM( dt ),
                                       DomainPartition & GEOSX_UNUSED_PARAM( domain ) )
{}
```
which would allow me to entirely remove `WellSolverBase::implicitStepComplete`